<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QMainWindow" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Guitarra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Viento</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Teclado</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Agregar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Instrumento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_VerStock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver Stock</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Marca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_Precio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Precio</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_CantCuerd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de Cuerdas</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_CantTecl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de Teclas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Marca">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Precio">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_CantCuer">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_CantTecl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_Metal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Metal Usado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Metal">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
